_commands/basics/id.md

```diff
@@ -16,9 +17,33 @@ ToDo: Needs completion
 
 #### Example command
 
-##### Break it down
+to find a specific user's UID, add -u {username}. Moreover, id -G USERNAME will find all the groups a user belongs to. Force id to display the name of the UID or GID instead of ugly numbers by passing the -n options
+
+~~~ bash
+$id -ng username
+staff eecsstudents club1 club2
+~~~
 
 #### Example command
+To find all the groups associated with a user called exampleuser, enter:
+
+~~~ bash
+$ id exampleuser
+~~~
+
+sample output:
+
+~~~ bash
+uid = 0(exampleuser) gid=0(eecs) groups=1(computing), 1(hello),2(this), 3(is), 4(an), 5(example)
+~~~
 
-##### Break it down
+### Useful options:
```

Comment:
You should just be able to push your changes to your Github fork and the PR should update automatically, no?